plugins/modules/apt_rpm.py

```diff
@@ -33,7 +32,20 @@
     type: str
   update_cache:
     description:
-      - update the package database first C(apt-get update).
+      - Run the equivalent of C(apt-get update) before the operation. Can be run as part of the package installation or as a separate step.
+      - Default is not to update the cache.
+    type: bool
+    default: false
+  clean:
+    description:
+      - Run the equivalent of C(apt-get clean) to clear out the local repository of retrieved package files. It removes everything but
+        the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/.
+      - Can be run as part of the package installation (clean runs before install) or as a separate step.
+    type: bool
+    default: false
+  upgrade:
```

Comment:
Shouldn't this option be called `dist_upgrade` then? There is also a `apt-get upgrade` command, and I would expect `upgrade: true` to run `apt-get upgrade` and not `apt-get dist-upgrade`.

Reply:
Unlike Debian, the apt-rpm implementation does not recommend using apt-get upgrade.

Reply:
That's one more reason from my POV to call the option `dist_upgrade` and not `upgrade`.

Reply:
> That's one more reason from my POV to call the option `dist_upgrade` and not `upgrade`.

On what system? Why apt module does not have `dist_upgrade` parameter (this command uses as option for `upgrade`)?

Reply:
> On what system?

The module calls `apt-get dist-upgrade`, not `apt-get upgrade`. So the option should best be called `dist_upgrade` and not `upgrade`. Calling it `upgrade` is misleading.

> Why apt module does not have `dist_upgrade` parameter (this command uses as option for `upgrade`)?

Because a) this module is not the `apt` module, and b) the 'apt' module's `upgrade` parameter needs to be set to `dist` to do `apt-get dist-upgrade`, and not simply to `true`.